WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2262,3 +2262,21 @@ PrefRowCheckWidget          = padding:2px 0;
 PrefRowSelectWidget         = min-width:280px; width:auto; max-width:100%;
 PrefRowInlineTable          = padding:0 @PrefViewInnerSpacing@;
 CheckWidgetInputText        = padding-top:0; padding-bottom:0; height:auto; line-height:1;
+
+######################################
+#   WhiteBlackList SPECIFIC STUFF
+######################################
+
+WhiteBlackViewInnerSpacing                 = @SkinViewInnerSpacing@
+TrustedAddrPrefHeader                      = margin-bottom:@SkinWrapperPadding@
+TrustedAddrHeader                          = margin-bottom:@SkinWrapperPadding@;
+WhiteBlackViewInput                        = width:413px; border-width:1px; @MediumRoundCorners@; padding-left:@WhiteBlackViewInnerSpacing@
```

Comment:
I would suggest not to hard code width of input. Instead set required width to parent. And this width should be relative to parent, so that this widget can fit almost anywhere.